### PR TITLE
fixes: get a releasable build

### PR DESCRIPTION
**Pull request by bonomat**
... more to come...

### CODE REVIEW

**Comment by 1010Tom** [comment]
don't merge yet. Being tested on my fork atm https://github.com/bonomat/10101/actions/runs/5888957442/job/15971187106

**Comment by 1010Tom** [comment]
Build complete, resulting version works on iphone :)

**Review by holzeis — APPROVED** [review]
LGTM 👍 if its working on your fork!

**Pull request merged**
